shopfloor_delivery_shipment/services/delivery_shipment.py

```diff
@@ -593,46 +582,87 @@ def _data_for_content_to_load_from_pickings(self, shipment_advice):
             pickings_partially_loaded = loaded_lines.picking_id
             domain += [("picking_id", "in", pickings_partially_loaded.ids)]
         move_lines = self.env["stock.move.line"].search(domain)
-        return self._prepare_data_for_content(move_lines)
+        return self._prepare_data_for_content(move_lines, False)
```

Comment:
I would make this arg named, so we can understand what this False means.
Also, I would remove the plural in the name.
```suggestion
        return self._prepare_data_for_content(move_lines, group_by_sale=False)
```
I don't know what default value is the most relevant here. Up to you.

Reply:
done